python/setup.py

```diff
@@ -23,7 +23,11 @@
 
 
 def config_cython():
-    """Try to configure cython and retyurn cython configuration"""
+    """Try to configure cython and return cython configuration"""
+    if os.name == 'nt':
+        print("WARNING: Cython is not supported on Windows, will compile without cython module")
+        return []
```

Comment:
a better way is to try cython build, catch the exception if it fails and do a non-cython build